js/apps/admin-ui/src/realm-settings/user-profile/UnmanagedAttributesTab.tsx

```diff
@@ -0,0 +1,141 @@
+import { useTranslation } from "react-i18next";
+import { KeycloakSpinner } from "../../components/keycloak-spinner/KeycloakSpinner";
+import { useUserProfile } from "./UserProfileContext";
+import {
+  FormGroup,
+  Grid,
+  GridItem,
+  PageSection,
+  Switch,
+} from "@patternfly/react-core";
+import { HelpItem } from "ui-shared";
+import { UnmanagedAttributePolicy } from "@keycloak/keycloak-admin-client/lib/defs/userProfileMetadata";
+import { FormAccess } from "../../components/form/FormAccess";
+import { useEffect, useState } from "react";
+
+export const UnmanagedAttributesTab = () => {
+  const { config, save } = useUserProfile();
+  const { t } = useTranslation();
+  const [enableUnmanaged, setEnableUnmanaged] = useState(false);
+
+  useEffect(() => {
+    setEnableUnmanaged(config?.unmanagedAttributePolicy != undefined);
+  }, [config]);
+
+  if (!config) {
+    return <KeycloakSpinner />;
+  }
+
+  function setPolicy(policy?: UnmanagedAttributePolicy) {
+    save({
+      ...config,
+      unmanagedAttributePolicy:
+        config?.unmanagedAttributePolicy === policy
+          ? UnmanagedAttributePolicy.enabled
+          : policy,
+    });
+  }
+
+  return (
+    <PageSection variant="light">
+      <FormAccess isHorizontal role="manage-realm">
+        <FormGroup
+          label={t("enableUnmanaged")}
+          fieldId="kc-up-unmanaged-attribute-policy-enableUnmanaged"
+          labelIcon={
+            <HelpItem
+              helpText={t("enableUnmanagedHelpText")}
+              fieldLabelId="enableUnmanaged"
+            />
+          }
+          hasNoPaddingTop
+        >
+          <Switch
+            id="kc-up-unmanaged-attributes-enableUnmanaged"
+            data-testid="up-unmanaged-attributes-enableUnmanaged"
+            value={enableUnmanaged ? "on" : "off"}
+            label={t("on")}
+            labelOff={t("off")}
+            isChecked={enableUnmanaged}
+            onChange={(value) => {
+              setPolicy(value ? UnmanagedAttributePolicy.enabled : undefined);
+            }}
+            aria-label={t("enableUnmanaged")}
```

Comment:
There should be no need to add an `aria-label` here, the control is already labeled by the `FormGroup`.

Reply:
We no longer have this tab.